components/booking/src/init.spec.js

```diff
@@ -1,16 +1,16 @@
-let schedulerTestComponent;
+let bookingTestComponent;
 let availabilityTestComponent;
 const frozenDateTime = () => new Date(2021, 11, 15, 9, 0, 0);
 beforeEach(() => {
   cy.clock(frozenDateTime().getTime(), ["Date"]);
-  cy.visit("/components/scheduler/src/index.html");
+  cy.visit("/components/booking/src/index.html");
 
-  // scheduler component
-  cy.get("nylas-scheduler").should("exist");
-  cy.get("nylas-scheduler").then((element) => {
+  // booking component
+  cy.get("nylas-booking").should("exist");
+  cy.get("nylas-booking").then((element) => {
```

Comment:
For consistency, I think the component ID should be updated in dynamoDB to `"test-booking"`. I think the `type` prop may need to be changed to "booking" as well 🤔